app/frontend/wallet/account.ts

```diff
@@ -315,6 +317,8 @@ const Account = ({
   }
 
   function isAccountUsed(): Promise<boolean> {
+    // in case the wallet is not shelley compatible we consider the
```

Comment:
this deserves a TODO comment instructing to decouple shelley compatibility from usedness as the current state is very confusing as discussed before